readme.md

```diff
@@ -0,0 +1,237 @@
+As far as it is reference implementation we will accent on developer's usage. You can connect to domain by:
+1) serial over USB
+2) SSH over ethernet
+
+### Serial over USB
+You need to connect your PC to board by USB-microUSB cable.
+
+Use your favorite serial console (minicom, cu etc).
+
+Set port to: 115200, 8N1, software flow control.
+
+Connect your PC to board's micro USB connector named "Debug serial". If needed - use detailed and labeled photo in wiki https://github.com/xen-troops/meta-xt-prod-devel/wiki
+
+Turn on board and see messages are running.
+
+### SSH over ethernet
+At first determine IP that is assigned to board.
+
+If serial already works - you can login to domD and check:
+```
+dom0 login: root
+# xl console DomD
+domd login:root
+# ifconfig
+```
+Or check your DHCP server. For example:
+```
+systemctl status isc-dhcp-server
+```
+
+Connect to board
+```
+ssh root@<board_ip>
+```
+After that you will be logged straight into DomD.
+Also you can connect to DomU, just use port 23:
+```
+ssh -p 23 root@<board_ip>
+```
+
+### What to do "on board"
+Use `root` to login to dom0 or domD (yes, it's just demo, and yes, we will close this security hole).
+
+Main program inside dom0 is `xl`. You can use
+```
+xl list                            ## to see list of active domains
+xl console DomD                    ## to switch to DomD's console (press Ctrl-5 to return to Dom0)
+xl destroy DomU
+xl create -c /xt/dom.cfg/domu.cfg  ## to create domain using specified config
+```
+Detailed manual for `xl` you can find at https://xenbits.xen.org/docs/unstable/man/xl.1.html
+
+DomD is quite regular Linux so you can use lots of regular Linux tools to do lots of regular Linux things.
+
+Also pay attention that domD is home for backend drivers for guest domains like DomU or DomA. This means that domU/domA is working with real hardware using drivers in domD. And if you stop network in domD, domU/domA will be isolated from world as well.
+
+If you want to return to dom0 - use `Ctrl-5`.
+
+
+## How to build
+Due to complex dependecies between different projects, you need to use special python script to build final product. Script will pull required layers and set configuration files.
+So you need to get script from github and edit few lines in config.
+Either clone repo
+```
+git clone https://github.com/xen-troops/build-scripts.git
+```
+or just download and unpack archive
+```
+wget https://github.com/xen-troops/build-scripts/archive/master.zip
+```
+Folder contains two python files (script itself), and configs for different products and specific cases.
+
+Create `local-devel-u.cfg` with following content:
+```
+[git]
+xt_history_uri = ""
+xt_manifest_uri = git@github.com:xen-troops/meta-xt-products.git
+[path]
+workspace_base_dir = /<your_storage>/work_devel/build
+workspace_storage_base_dir = /<your_storage>/storage
+workspace_cache_base_dir = /<your_storage>/work_devel/tmp
+[local_conf]
+XT_GUESTS_INSTALL = "domu"
+XT_GUESTS_BUILD = "domu"
```

Comment:
XT_RCAR_EVAPROPRIETARY_DIR is missing? I am asking because **--product devel** is mentioned in a few examples which implies the binaries to be located somewhere

Reply:
Info regarding XT_RCAR_EVAPROPRIETARY_DIR is added.
You are right, seems that text has some kind of conflicts regarding build with/without binaries. Need to re-read and rework.